dora/core/common/src/main/java/alluxio/uri/UfsUrl.java

```diff
@@ -135,6 +142,39 @@ public String getAuthority() {
     public List<String> getPathComponents() {
       return mPathComponents;
     }
+
+    /**
+     * Normalize the path component of the {@link UfsUrl}, by replacing all "//" with
+     * "/", and trimming trailing slash from non-root path.
+     * It is inspired by AlluxioURI.normalizePath(String).
+     *
+     * @param path the path to normalize
+     * @return the normalized path
+     */
+    public static String removeRedundantSlashes(String path) {
```

Comment:
Does this have to be public?

Reply:
It is originally in PathUtils, I'm sorry for forgetting modify the function signature.